### DISCUSSION TITLE
Bad UX - backend - if views are empty

### DISCUSSION THREAD

**Discussion by srishty-07** [joomla/joomla-cms#32956] [2. Cold Features]
### Problem identified
on going on system>language>language-overrides ,there is a text bar with 'no matching result' written on it without even searching something, the same issue happens on other pages also example component>banner, tracks .
It occurs when  there is no data  on the page.
 
![image](https://user-images.githubusercontent.com/70388131/113267931-7ce52680-92f4-11eb-9915-0a4c87031721.png)
![image](https://user-images.githubusercontent.com/70388131/113268068-a0a86c80-92f4-11eb-9b9b-d085cb142fdc.png)

### Proposed solution
Instead of info-alert nothing should be there.

**Comment by brianteeman** [comment]
Why should there be nothing?

**Comment by srishty-07** [comment]
> Why should there be nothing?

according to me this info-alert box should come only when a user search something and don't get the required result ,                             it will prevent confusion.

**Comment by chmst** [comment]
This is right. We could differ between "There is no override yet" and "no matching result".

**Comment by richard67** [comment]
> This is right. We could differ between "There is no override yet" and "no matching result".

Yes, but then it should be done consistently everywhere, not only for the overrides, i.e. "There is no %s yet" or something like that in all list views where no item exists yet, and "No matching result" if a search or filtering gives no result.

I can imagine that people think that the word "matching" in the message refers to a search or filtering, so I can understand the motivation behind this issue (which I would say is an RFC).

Personally I can live with it like it is, but that's maybe because of my developer point of view.

**Comment by brianteeman** [comment]
It would not be a simple change to make across all the uses. This is because a lot of the views are actually filtered even if you dont realise

**Comment by richard67** [comment]
> It would not be a simple change to make across all the uses. This is because a lot of the views are actually filtered even if you dont realise

Exactly that's why at the end I can live with like it is now.

**Comment by chmst** [comment]
It is worth a discussion, but surely not in 4.0. 
So I will move it to discussion.
